README.md

```diff
@@ -38,12 +38,20 @@ cd lean-quickstart
 
 ### Quickly startup various nodes as a local devnet
 
+**Using shell scripts (quick local setup):**
 ```sh
 NETWORK_DIR=local-devnet ./spin-node.sh --node all --generateGenesis --popupTerminal
 ```
 
+**Using Ansible (recommended for production/remote):**
+```sh
+./ansible-deploy.sh --node all --network-dir local-devnet --generate-genesis
```

Comment:
we should adopt the same spin-node.sh and its arg format instead of having a different entry point, may be add a field in validator-config.yaml at top level to indicate this: `local` or `ansible`

also the steps should be deeply integrated with spin-node, so instead of doing something local, ansible does it remote